### DISCUSSION TITLE
v5的页面搜索使用体验不好

### DISCUSSION THREAD

**Discussion by YuzKio** [antvis/G6#6300] [General]
https://g6-next.antv.antgroup.com/
输入框键入快的时候会卡顿很久，搜索很慢，而且有时候搜索会导致页面卡死。

**Comment by resetsix** [comment]
参考 https://github.com/antvis/G6/issues/6281

实际原因：v5 文档搜索时，会发起至少 5k+ 海量 http 请求（跟搜索内容无关，无论搜索什么内容都是固定请求），导致浏览器响应卡顿。v4 文档搜索功能正常，只有 v5 文档存在异常。

初步判断：dumi 版本（或配置问题）导致。

<img width="523" alt="图片" src="https://github.com/user-attachments/assets/c408374d-d11b-4937-95c3-a333082dbe98">

**Comment by Aarebecca** [comment]
目前正在等待 dumi 更新，修复后会重新部署官网
